Release note for CIE-5477 (#3867)
SoftwareAG/c8y-docs: content/change-logs/edge/edge-2025.0.8-uninstallation.md

---
date: 2024-01-09

> title: Deleting the Edge installation now removes all volumes from kubernetes

Reviewer: ```suggestion
title: Deleting the Edge installation now removes all volumes from Kubernetes
```

change_type:
  - value: change-VSkj2iV9m
    label: Fix
product_area: Edge
component:
  - value: component-g7hnfbu4J
    label: Edge
build_artifact:
  - value: tc-nJH2U7g3u
    label: edge
ticket: CIE-5477
version: 2025.0.8
---

> The uninstallation procedure for Edge on a self-managed Kubernetes cluster includes the deletion of the Edge Custom Resource. This was not deleting the MongoDB volume, leaving behind a potentially space-consuming chunk of tenant data on disk. This sometimes caused subsequent fresh installations to be using old tenant data, resulting in various sorts of malfunction. This has now been fixed, and the uninstallation procedure cleans up all volumes associated with that Edge instance.

Reviewer: ```suggestion
The uninstallation procedure for Edge on a self-managed Kubernetes cluster includes the deletion of the Edge Custom Resource. Previously, this did not delete the MongoDB volume, leaving behind a potentially space-consuming chunk of tenant data on disk. This sometimes caused subsequent fresh installations to use old tenant data, resulting in various sorts of malfunctions. This issue has now been fixed, and the uninstallation procedure cleans up all volumes associated with that Edge instance.
```
